packages/augur-ui/src/modules/auth/actions/login-with-fortmatic.ts

```diff
@@ -10,13 +10,13 @@ import { getNetworkId } from 'modules/contracts/actions/contractCalls';
 import { windowRef } from 'utils/window-ref';
 import { AppState } from 'store';
 
-const getFormaticNetwork = (networkId: string): false | string   => {
+const getFormaticNetwork = (networkId: string): string   => {
```

Comment:
Can this be moved out to a helper to convert networkId to network bame

Reply:
There's already a mapping for this, yah